app/views/publish/index.html.erb

```diff
@@ -42,13 +42,12 @@
       <%# span class="name by"><%= t('publish.test.by', name: 'C. Smith') %></span %>
     <%# /p %>
 
-    <span>
-      <%= PublishServicePresenter.hostname_for(deployment_environment: 'production', view: self) %>
-    </span>
+    <p><%= PublishServicePresenter.hostname_for(deployment_environment: 'production', view: self) %></p>
 
+    <%# TODO: Hardcoded 'firstpublish' (see comment above) %>
     <%= form_for(@publish_service_creation_production,
         url: publish_index_path(service.service_id),
-        html: { id: 'publish-form-live', class: 'publish-form' }) do |f| %>
```

Comment:
@tomas-stefano 
Second hardcoded value that controls things for Production environment. Same thing as previous comment applies.